doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,141 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit image to function as a bluetooth speaker
```

Comment:
Rather than a section head, make this an intro sentence:

The instructions below show you how to build a bluetooth speaker using an Intel Joule development board running intel-iot-refkit image.